templates/observability/what-is-observability.html

```diff
@@ -188,10 +188,19 @@ <h3 class="p-heading--4">Making the most out of your metrics, logs, distributed
 </section>
 
 <section class="p-strip">
-    <div class="row">
-        <div class="col-8">
+      <div class="row">
+        <div class="col-12">
             <h2>Telemetry context</h2>
             <p>An often underrated aspect of telemetry is its context; that is, which system is the telemetry coming from, where is that system running and, by extension, what is the relation (or lack thereof) between this piece of telemetry and others. You could think of this as correlating telemetry with the topology of the monitored systems.</p>
+            {{ image (
+                url="https://assets.ubuntu.com/v1/485bb400-telemetry+types-substrates+and+context-02.svg",
+                alt="Types of telemetry at the different substrate levels, namely bare metal, virtualization, operating system, containers and serverless, applications and end-user devices. Metrics and logs are prevalent across all the substrates; end-user monitoring and distributed tracing are at application level, production profiling is found between the application and operating system substrates.",
+                width="925",
+                height="488",
+                hi_def=True,
+                loading="auto"
```

Comment:
We should set this to lazy, it seems to load it right away on page load. Also, what do you think about this for spacing above and below? I've borrowed it from the image above this one.
```suggestion
                loading="lazy",
                attrs={"style": "padding: 2rem 0;", "class":"u-hide--small"},
```